aiinfra-cluster/main.tf

```diff
@@ -46,8 +46,17 @@ locals {
       "content"         = "${var.startup_command}"
     }
   ] : []
+
+  install_ops_agent = !var.disable_ops_agent ? [
+    {
+      "type"        = "shell"
+      "destination" = "install_cloud_ops_agent.sh"
+      "source"      = "${path.module}/installation_scripts/install_cloud_ops_agent.sh"
+    }
+  ] : []
   
-  vm_startup_setup      = concat(local.ray_setup, local.startup_command_setup)
+  vm_startup_setup      = concat(local.ray_setup, local.startup_command_setup, local.install_ops_agent)
```

Comment:
this order actually defines the order of the script execution when the vm starts up. I think lets move `install_ops_agent` before `startup_command_setup`

Reply:
Updated.